README.rst

```diff
@@ -216,31 +218,46 @@ IP addresses usage:
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - 192.168.0.101
-      - 192.168.0.102
-      - 192.168.0.103
-
+    clusters:
+      - nodes:
+        - 192.168.0.101
+        - 192.168.0.102
+        - 192.168.0.103
+      - nodes:
+        - 192.168.1.101
+        - 192.168.1.102
+        - 192.168.1.103
+        
 Minion id usage:
 
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - minion1
-      - minion2
-      - minion3
+    clusters:
+      - nodes:
+        - minion1
+        - minion2
+        - minion3
+      - nodes:
+        - minion4
+        - minion5
+        - minion6
 
 Mixed usage (IP, minion id, fqdn):
 
 .. code:: yaml
 
   zookeeper:
-    nodes:
-      - 192.168.0.101
-      - minion2
-      - zookeeper3.mysite.com
-      
+    clusters:
+      - nodes:
+        - 192.168.0.101
+        - minion2
+        - zookeeper3.cluster1.mysite.com
+      - nodes:
+        - 192.168.1.101
+        - minion-hostname5
+        - zookeeper3.cluster2.mysite.com
```

Comment:
Then the first one will be used.

For example:

```
cluster1:
  - 192.168.0.101
  - 192.168.0.102
  - 192.168.0.103
cluster2:
  - 192.168.0.101
  - 192.168.1.104
  - 192.168.1.105
```

in this case `192.168.0.101` is used both in `cluster1` and `cluster2`. If you try to deploy such clusters, then the host `192.168.0.101` will connect to the first cluster. I don't know, but second cluster either won't work or will work, but hosts `192.168.1.104` and `192.168.1.105` will think that `192.168.0.101` is dead.

I know that zookeeper let us run multiple zookeeper servers on a single machine (by using different ports) but it is used only for testing.
I pointed in `readme` that we cannot use the same minion for two different clusters.